alerts/openshift-container-storage-operator/CephOSDCriticallyFull.md

```diff
@@ -0,0 +1,38 @@
+# CephOSDCriticallyFull
+
+## Meaning
+
+Utilization of back-end storage device (OSD) has crossed 80%.
+Immediately free up some space or expand the storage cluster or contact support.
+
+## Impact
+
+One of the OSD Storage size has crossed 80% of the total capacity. Expand the cluster immediately.
+
+## Diagnosis
+
+Alert message will have enough information about the underlying failure.
+It should show the name of 'ceph-daemon', 'device-class' and the 'host-name'.
+
+A sample alert message is provided below,
+
+    Utilization of storage device <ceph-daemon-name> of device_class type <device-class-name> has crossed 80% on host <host-name>. Immediately free up some space or add capacity of type <device-class>.
+
+## Mitigation
+
+### Delete data
+
+The customer may delete data and the cluster will resolve the alert through self healing processes.
+
+### Expand the storage capacity
+
+Customer may assess their ability to expand. Here are some points,
+
+**Current Storage Size < 1TB**:  
+The customer may increase capacity via the addon and the cluster will resolve the alert through self healing processes.
+
+**Current Size itself is 1TB**:
+Please contact your dedicated customer care support.
+
+[gather_logs](helpers/gather_logs.md)
```

Comment:
wrong filename

Reply:
May be the files were added later (by another PR). Can you please check?

Reply:
The files are there since two months ago
![image](https://github.com/openshift/runbooks/assets/1820049/15490df8-734b-4eb7-8bb4-ae95e1bc45c1)

Reply:
Ack. Corrected...